.github/workflows/pr-description-update.yml

```diff
@@ -15,7 +15,8 @@ concurrency:
 jobs:
   update_pr:
     # we don't want this to run on breaking change PRs or release please PRs
-    if: contains(github.head_ref, 'release-please') != true && contains(github.head_ref, 'breaking-release') != true
+    if: contains(github.head_ref, 'release-please') != true && contains(github.head_ref, 'breaking-release') != true && ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
does the thing expand as expected? the other parts of the if condition has different syntax